src/OrbitGl/App.cpp

```diff
@@ -2661,6 +2661,8 @@ bool OrbitApp::IsLoadingCapture() const {
   return data_source_ == orbit_client_data::CaptureData::DataSource::kLoadedCapture;
 }
 
+bool OrbitApp::IsDataStreamingActive() const { return IsCapturing(); }
```

Comment:
This reminds me that we might have methods shared by different modules. If we consider defining different `AppInterface`s for different modules, should we extract those shared funtionality into a `CommonAppInterface`?

Reply:
I've now just named them the same in both interfaces. I think that in the long run, the DataViews should not define interfaces, but rather rely on the ones defined in other places... but for now, I'd just leave it as is.

Reply:
I agree 100%. This is definitely a temporary state. DataViews (or later ItemModels) should get its data via Setters and should send events using signals. No coupling or interfaces required apart from the item data type.